superset-frontend/package.json

```diff
@@ -201,6 +201,7 @@
     "rimraf": "^6.0.1",
     "rison": "^0.1.1",
     "scroll-into-view-if-needed": "^3.1.0",
+    "simple-zstd": "^1.4.2",
```

Comment:
@kgabryje shouldn't this be in `devDependencies`

Reply:
It should, I moved it to dependencies to silence eslint errors. It shouldn't really affect the entry point sizes though, as webpack will put this package in a chunk that's not downloaded on any page